### PR TITLE
about me page

### PR DESCRIPTION
**Pull request by cindytung**
- Used Montserrat for all text
- Created section + page header elements so text can be consistent across all pages
- Merged navbar in

TODO(Cindy):
- Make navbar responsive (will make a PR for this soon)
- Do home/alumni page

### CODE REVIEW

File: components/Officers.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      name: 'Lydia Chen',
+      position: 'Recruiting Co-Chair / Media Officer',
+      email: 'lydiachen@berkeley.edu',
+      image: './assets/images/officer-test.png',
+    },
+  ];
+
+  const officerElements = () => (
+    <div className="officers">
+      {this.people.map((data) => {
```
Comment (jubearsun) [inline components/Officers.js]: nit: You can do arrow return
```js
this.people.map((data) => (
  <Photo data={people} key={data.name} />
));
```

File: components/photo.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+
+const Photo = (props) => (
```
Comment (jubearsun) [inline components/photo.js]: `class Photo extends React.Component`

**Review by jubearsun — CHANGES_REQUESTED** [review]

File: components/photo.js (hunk 2 above)
Comment (cindytung) [reply components/photo.js]: changed impl for Photo to stateless, so don't need to extend React.Component anymore

File: css/aboutme.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
+body {
```
Comment (jubearsun) [inline css/aboutme.scss]: Remove this line if there are no styles in it

File: css/aboutme.scss [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+#team {
```
Comment (jubearsun) [inline css/aboutme.scss]: The indents in this file are off, can you double check that they're 2 space indents?

File: css/aboutme.scss [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	width: 100%;
+}
+
+.officers, .coaches {
+	padding: 2% 12%;
+	text-align: center;
+
+	.officer__container, .coach__container {
+		position: relative;
+    margin-bottom: 2%;
```
Comment (jubearsun) [inline css/aboutme.scss]: Please fix indentation here and elsewhere

File: css/base.scss [hunk 6]
```diff
@@ -10,8 +12,41 @@
...
 body {
+  font-family: 'Montserrat', sans-serif;
   position: relative;
   width: 100%;
   height: 100%;
 }
 
+.page__header {
+	font-size: 45px;
+	text-align: center;
```
Comment (jubearsun) [inline css/base.scss]: Please fix indentation here and elsewhere.

File: pages/about.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+// import { Link } from 'react-router';
```
Comment (jubearsun) [inline pages/about.js]: Remove this comment if unused

**Review by jubearsun — CHANGES_REQUESTED** [review]

**Pull request merged**